lib/commands/run.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
 	run: function(options) {
 		if (configExists()) {
 			runTask('build', ['watch', 'server'], {
+				skipBundle = options.skipBundle,
```

Comment:
Did this not throw a syntax error when you tested? You can't set object literal properties this way.